# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/gtest/TestVideoTrackEncoder.cpp
# Commit: 48eaa84c206d
# Full Hash: 48eaa84c206dbdda637c57b55ddbd07e975ba103
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2020-06-23 16:15:06
# Regressor Bug: 1637500
# File Overlap Count: 2
# Description:
#   Bug 1637500 - P2. Rename methods as they are not always dealing with "threads". r=froydnj
#   
#   Before P1, GetCurrentThreadSerialEventTarget would have always returned the same data as NS_GetCurrentThread, making the comment incorrect Now it will properly return the running TaskQueue if any.
#   
#   This change of name more clearly exposes what they are doing, as we aren't always dealing with threads directly; but a nsISerialEventTarget
# ==============================================================================

diff -r 00687d06a8a6 -r 48eaa84c206d dom/media/gtest/TestVideoTrackEncoder.cpp
--- a/dom/media/gtest/TestVideoTrackEncoder.cpp	Tue Jun 23 05:02:39 2020 +0000
+++ b/dom/media/gtest/TestVideoTrackEncoder.cpp	Tue Jun 23 05:05:36 2020 +0000
@@ -36,7 +36,7 @@
 class MockDriftCompensator : public DriftCompensator {
  public:
   MockDriftCompensator()
-      : DriftCompensator(GetCurrentThreadEventTarget(), VIDEO_TRACK_RATE) {
+      : DriftCompensator(GetCurrentEventTarget(), VIDEO_TRACK_RATE) {
     ON_CALL(*this, GetVideoTime(_, _))
         .WillByDefault(Invoke([](TimeStamp, TimeStamp t) { return t; }));
   }